color/README.md

```diff
@@ -26,6 +26,46 @@ See https://linebender.org/blog/doc-include/ for related discussion. -->
 
 Color is a Rust crate which implements color space conversions, targeting at least CSS4 color.
 
+## Scope and goals
+
+Color in its entirety is an extremely deep and complex topic. It is completely impractical
+for a single crate to meet all color needs. The goal of this one is to strike a balance,
+providing color capabilities while also keeping things simple and efficient.
+
+The main purpose of this crate is to provide a good set of types for representing colors,
+along with conversions between them and basic manipulations, especially interpolation. A
+major inspiration is the CSS Color Level 4 draft spec; we implement most of the operations
+and strive for correctness.
+
+Simplifications include:
+  * Always using `f32` to represent component values.
+  * Only handling 3-component color spaces (plus optional alpha).
+  * Choosing a fixed, curated set of color spaces for dynamic color types.
+  * Choosing linear sRGB as the central color space.
+  * Keeping white point implicit.
+
+A number of other tasks are out of scope for this crate:
```

Comment:
I think it is worth saying that some of these may be added as other crates within the `color` ecosystem (of course, some can not be).